library/core/src/iter/adapters/zip.rs

```diff
@@ -413,6 +413,22 @@ where
 {
 }
 
+#[stable(feature = "exact_size_zip_repeat", since = "1.91.0")]
```

Comment:
Master just changed to 1.92, and in general you can't be sure when things will merge, so use this placeholder:
```suggestion
#[stable(feature = "exact_size_zip_repeat", since = "CURRENT_RUSTC_VERSION")]
```